### ISSUE TITLE
Waves effect is blurry

### ISSUE THREAD

**Issue by sodbileg** [Dogfalo/materialize#816]
When I add `waves-effect waves-block waves-light` class to an element and click on that element, a waves effect is so blurry. I recorded my screen, and the link is on below. What should I do?

[Video](https://www.dropbox.com/s/3iy2714i74cjjpb/Untitled.mov?dl=0)

**Comment by Dogfalo** [comment]
Are you doing any transforms to the element?

**Comment by sodbileg** [comment]
No, I'm not. Is it possible to depend on jQuery version? I'm using it with WordPress.

**Comment by sodbileg** [comment]
So I found the issue. It's not related to jQuery nor WordPress. I changed the width of `.container` to 80% in large-and-up media query like below.

```
@media #{$large-and-up} {
    .container {
        width: 80%;
    }
}
```

And when I commented out this code, it works without any blur. So is there any possible way to change the width of container without getting blurry waves effect on images?

**Comment by Dogfalo** [comment]
Fixed by 53eaec41f9f29c9775d1fbf290c2f5a060fdebda
